ng-dev/release/notes/templates/github-release.ts

```diff
@@ -20,14 +20,8 @@ _%>
   for (const group of asCommitGroups(breakingChanges)) {
 _%>
 ### <%- group.title %>
-
-<%_
-    for (const commit of group.commits) {
-_%>
-<%- commit.breakingChanges[0].text %>
```

Comment:
@josephperrott While seeing this: do you know what happens if I have multiple breaking changes in a single commit? seems like something we should fix?

Reply:
You can see https://github.com/angular/angular-cli/releases/tag/13.0.0-next.0

Reply:
I have never considered putting multiple `BREAKING CHANGE` blocks in a single commit, I suppose we should handle that case

Reply:
Oh I now see what @devversion meant.

Reply:
yeah, it's something we did do in the COMP repo at some point IIRC. It seems reasonable for me